src/AutocompleteModal/AutoComplete.tsx

```diff
@@ -74,10 +73,7 @@ const AutoComplete = ({
   });
 
   return (
-    <div
-      className={cc(["nds-autocompleteModal-autocomplete", "rounded--all"])}
-      {...getComboboxProps()}
-    >
```

Comment:
This is the biggest change. The ARIA spec for the combobox pattern changed from ARIA 1.1 to 1.2

The recommended pattern is now to only apply the aria attributes to the input itself rather than a common parent between the input and the menu.

Downshift removed `getComboboxProps` as a result because it's no longer needed.